### DISCUSSION TITLE
Encrypted (PGP) Email Processing

### DISCUSSION THREAD

**Discussion by jobr97** [jonaswinkler/paperless-ng#1067] [Feature Requests]
Hi, thanks for this awesome opensource project.

Since I mainly scan all my documents with an app on my phone I added an email account to my paperless instance. However I don't really feel comfortable sending all these often sensitive documents over unencrypted email. So my feature request is the implementation of PGP-encrypted email. 

The way I imagine it, it would be an optional setting in the email account where I can generate or import a key-pair and export the public-key. When a new encrypted email is received, paperless tries to decrypt it using the private-key.

**Comment by Coo-ops** [comment]
Apologies for the extra email chatter. I came to +1 this one because I feel like it can provide a tangible privacy increase for all users of the email ingest function - whether you're self hosted (on a VPS, the only sensible way to self host email), or using a third party service such as Gmail or Hotmail/Live.
